src/containers/navigation-icons/AccountIcon.tsx

```diff
@@ -58,19 +58,23 @@ const AccountIcon: FC<AccountIconProps> = ({ openMenu }) => {
   }, [photo, statePhoto])
 
   if (loading) {
-    return <Avatar sx={styles.accountIcon} />
+    return (
+      <UserAvatar firstName='' lastName='' src='' sx={styles.accountIcon} />
```

Comment:
I noticed that both firstName and lastName have a default value of an empty string and are optional props. Please avoid explicitly passing them here